### ISSUE TITLE
Error with destructuring array within `if` condition with no braces

### ISSUE THREAD

**Issue by brettz9** [Rich-Harris/buble#152]
Getting an "Unexpected token" error with code such as the following:

```js
export function replaceRange(from, to) {
  if (from) [from, to] = [to, from]

}
```

The error doesn't occur if braces are present.

**Comment by marijnh** [comment]
Specifically, the generated code looks like this:

```
export function replaceRange(from, to) {
  var assign;

  if (from) { (assign = [to, from], from = assign[0], to = assign[1] })

}
```

Which has mismatched brackets (the paren after the destructuring expression ended up before the brace)

**Comment by marijnh** [comment]
See PR #153

**Comment by adrianheine** [comment]
Fixed with 8117b6902507374a2a88da3e763eda3e37387869.
